src/go/rpk/pkg/cli/cmd/debug/bundle_all.go

```diff
@@ -32,5 +32,9 @@ func executeBundle(
 	int,
 	time.Duration,
 ) error {
-	return errors.New("rpk debug bundle is unsupported on your operating system")
+	return errors.New("rpk debug bundle is not supported on your operating system")
+}
+
+func processFilepath(fs afero.Fs, path string, isFlag bool) (finalPath string, err error) {
```

Comment:
I think this needs to be changed to `determineFilepath` as well (non-linux build files)